src/gui/editors/PianoRoll.cpp

```diff
@@ -3918,31 +3918,33 @@ int PianoRoll::quantization() const
 
 void PianoRoll::quantizeNotes()
 {
-	NoteVector notes = getSelectedNotes();
+	if( m_pattern != NULL ) {
```

Comment:
While this definitively works I'd propose to use the same style like in `PianoRoll::cutSelectedNotes()`:
```C++
if( ! hasValidPattern() )
{
	return;
}
```
This make it more obvious what's tested and also keeps the indentation level of the method down.